echopype/convert/set_groups_ek80.py

```diff
@@ -194,7 +196,8 @@ def set_sonar(self) -> xr.Dataset:
                     ["channel"],
                     list(self.parser_obj.config_datagram["configuration"].keys()),
                     self._varattrs["beam_coord_default"]["channel"],
-                )
+                ),
+                "beam_group": beam_groups_coord["beam_group"],
```

Comment:
Thanks for catching this! Looking at my source code, I think I dropped it when resolving the conflict with your frequency > channel PR. Here's what I had, using the ** pattern I use elsewhere. It's cleaner, though less explicit:
```suggestion
                **beam_groups_coord,
```

Reply:
@emiliom I like your suggestion more. `**beam_groups_coord` is cleaner and is more consistent with the rest of the code.